__tests__/index.test.ts

```diff
@@ -0,0 +1,68 @@
+import request from 'supertest';
+import express from 'express';
+
+const app = express();
+
+import { generateCronExpression, generateReadableExpression } from '../src/cronGenerator';
+
+app.get('/v1/generate', (req, res) => {
```

Comment:
Do not duplicate server code! (remember, DRY (don't repeat yourself)

Refer to the guide mentioned in [jest docs](https://jestjs.io/docs/testing-frameworks#expressjs).